### DISCUSSION TITLE
Battery Drain

### DISCUSSION THREAD

**Discussion by marciolabio** [artginzburg/MiddleClick#71] [Q&A]
Hi, I'm on MBP M1 Pro and I'm having battery drain issues. I restarted in safe mode and everything seemed to work fine, then I tried to open one by one the third party apps I use. The second one was middleclick and it seems like my laptop is getting warmer since it's active and the battery is draining extremely fast. 

How can I solve it?

**Comment by artginzburg** [comment]
If you open `Activity Monitor` > `Energy`, what is the `Energy Impact` of MiddleClick?
